tests_new/integration_tests/modules/mlstudio/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from integration_tests.core.stack.utils import check_stack_ready
+from integration_tests.modules.mlstudio.mutations import create_smstudio_user, delete_smstudio_user
+from integration_tests.modules.mlstudio.queries import get_smstudio_user
+
+
+@pytest.fixture(scope='session')
+def smstudio_user1(session_id, client1, persistent_env1):
+    env_uri = persistent_env1.environmentUri
```

Comment:
Don't we have to ensure the `create_environment` API call has `{'key': 'mlstudioEnabled', 'value': 'true'},` as part of parameters?

Otherwise we wouldn't be able to spin up mew sm studio users

Reply:
yes you are right, I enabled it manually on my persistent environment and forgot to add it in code